test/test_transforms_v2_refactored.py

```diff
@@ -250,6 +250,9 @@ def _check_transform_v1_compatibility(transform, input, *, rtol, atol):
     with freeze_rng_state():
         output_v1 = v1_transform(input)
 
+    if all(isinstance(o, PIL.Image.Image) for o in [output_v2, output_v1]):
+        output_v2, output_v1 = [F.to_image(o) for o in [output_v2, output_v1]]
```

Comment:
> in case the output types mismatch, i.e. one tensor and one PIL.Image.Image

That should never happen because:

- we won't be modifying the v1 transforms
- we have a test that makes sure the types are preserved in v2

Honestly I would just simplify everything to a brutal:

```py
output_v1 = F.to_image(output_v1) 
output_v2 = F.to_image(output_v2) 
```

without any condition or check.